stash/src/service/FaucetService.ts

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
+import * as process from 'node:process'
+import { redis } from '../connector/RedisConnector.js'
+import logger from '../util/Logger.js'
+import { createWalletClient, http, publicActions } from 'viem'
+import { holesky } from 'viem/chains'
+import { privateKeyToAccount } from 'viem/accounts'
+import Gasp from '../Gasp.json' assert { type: 'json' }
+import { ForbiddenException } from '../error/Exception.js'
+
+const VERIFY_URL = 'https://api.hcaptcha.com/siteverify'
+//the two enums below you can use with token 10000000-aaaa-bbbb-cccc-000000000001 to have a test example of success
+const TEST_SECRET = '0x0000000000000000000000000000000000000000'
```

Comment:
Test secret and test sitekey are just until FE implements the widget and starts sending the real generated token. This now is to ease the testing of the BE.

Reply:
Removed test credentials